subprojects/core/src/integTest/groovy/org/gradle/execution/taskgraph/ParallelTaskExecutionIntegrationTest.groovy

```diff
@@ -454,4 +463,40 @@ class ParallelTaskExecutionIntegrationTest extends AbstractIntegrationSpec {
             failure.assertHasCause "BOOM!"
         }
     }
+
+    def "other tasks are not started when an invalid task task is running"() {
+        given:
+        withParallelThreads(2)
+
+        expect:
+        2.times {
+            executer.expectDocumentedDeprecationWarning("Property 'invalidInput' has @Input annotation used on property of type 'File'. " +
```

Comment:
Do you think it makes sense to have a `executer.expectTaskValidationWarning()` method, so we need to repeat fewer things?

Reply:
Maybe it would have made sense in the past, but given that we're going to remove them in 7.0 I don't think we should introduce it now.

Reply:
👍